core/server/common/src/main/java/alluxio/master/journal/checkpoint/CheckpointType.java

```diff
@@ -35,6 +35,10 @@
    * A RocksDB backup in .tar.gz format.
    */
   ROCKS(3, new TarballCheckpointFormat()),
+  /**
+   * A RocksDB checkpoint in .tar.gz format.
```

Comment:
oops, this wasn't supposed to slip into this PR. We never released back-based checkpoints, so we don't need to differentiate between them and checkpoint-based backups

backups and checkpoints are RocksDB concepts. Basically a backup is information set aside by the backup engine so that it can restore a database from the backup. Backups are full copies of all state. Checkpoints use hard links instead of copying to essentially fork a database. Unlike a backup, a checkpoint can be opened and used like a regular database